src/Components/Cart.jsx

```diff
@@ -1,8 +1,16 @@
 import { Dialog, Transition } from "@headlessui/react";
-import { XIcon } from "@heroicons/react/outline";
-import React, { Fragment } from "react";
+import { XIcon, ShoppingCartIcon } from "@heroicons/react/outline";
+import React, { Fragment, useState } from "react";
+import { useEffect } from "react";
 
 export default function Cart({ open, setOpen, cart, updateCart }) {
+  const [total, setTotal] = useState(0);
+  useEffect(() => {
```

Comment:
we might not need an `useEffect` and `useState` here, instead we can directly initialize the total amount and use it

Reply:
since every time cart changes it will trigger a re-render and the variable would re-initialize

Reply:
True. I have updated it. Thanks for the catch.